### PR TITLE
Add event creation form

### PR DESCRIPTION
**Pull request by gorkaj**

### CODE REVIEW

**Review by faliszewskii — APPROVED** [review]
LGTM

File: src/front/ticketer/lib/backend_communication/logic/event/communication_event.dart [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import 'package:ticketer/backend_communication/logic/communication.dart';
+import 'package:ticketer/backend_communication/model/event.dart';
+import 'package:ticketer/backend_communication/model/response_codes.dart';
+import 'package:tuple/tuple.dart';
+
+class EventCommunication {
+  static const String _createEndPoint = "/event/create";
+  Future<Tuple2<Response, ResponseCode>> create(Event body) async =>
+      await BackendCommunication().postCallAuthorized(
+          _createEndPoint, Token(Auth().getCurrentAccount!.token),
```
Comment (bjadczak) [inline src/front/ticketer/lib/backend_communication/logic/event/communication_event.dart:15]: Add check if user is not logged in it will throw an error

**Review by bjadczak — DISMISSED** [review]
Please fix before merging

File: src/front/ticketer/lib/backend_communication/logic/event/communication_event.dart (hunk 1 above)
Comment (gorkaj) [reply src/front/ticketer/lib/backend_communication/logic/event/communication_event.dart]: done

**Review by bjadczak — APPROVED** [review]
LGTM

**Pull request merged**